book/src/validator-stake.md

```diff
@@ -24,15 +24,17 @@ $ solana redeem-vote-credits ~/validator-config/stake-keypair.json ~/validator-v
 ```
 
 The rewards lamports earned are split between your stake account and the vote account according to the
-commission rate set in the vote account.
+commission rate set in the vote account.  Rewards can only be earned while the validator is up and 
+running.  Further, once staked, the validator becomes an important part of the network.  In order to 
+safely remove a validator from the network, first deactivate its stake.
 
 Stake can be deactivated by running:
 ```bash
 $ solana deactivate-stake ~/validator-config/stake-keypair.json ~/validator-vote-keypair.json
 ```
 
 The stake will cool down, deactivate over time.  While cooling down, your stake will continue to earn
-rewards.
+rewards.  Only after stake cool down is it safe to turn off your validator or withdraw it from the network.
```

Comment:
this "cooldown" s/b one word

Reply:
Also, can you add a sentence about how validators can tell if their stake is cooled down and safe to withdraw?

Reply:
I'd love to, but we don't have an API for that, do we?

Reply:
Ah. No, I'm getting ahead of myself. Nm

Reply:
well, in the case of single stake, single-validator, the live stakes API would show zero stake...

it's important for validators to know what their stake is before bailing and the current API gives them that...

Reply:
as for "safe to withdraw", just keep trying withdraw() until balance is zero ;)